RetailCoder.VBE/Refactorings/Rename/RenameModel.cs

```diff
@@ -76,5 +108,41 @@ public void PromptIfTargetImplementsInterface(ref Declaration target)
 
             target = interfaceMember;
         }
+
+        internal class Rewriters
```

Comment:
Why does rename know about rewriters? It should be able to just hook into the Rewriting API and do a single call instead of having to maintain this dictionary